src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelCopyRatioEmissionProbabilityCalculatorUnitTest.java

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.apache.commons.math3.util.FastMath;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.SexGenotypeData;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Unit tests for {@link CoverageModelCopyRatioEmissionProbabilityCalculator}
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class CoverageModelCopyRatioEmissionProbabilityCalculatorUnitTest extends BaseTest {
```

Comment:
Sure. As a matter of principle, asymptotic limits are both illuminating and good tests. However, it's difficult to do any assertion without having a theoretical bound on asymptotic formulas. Here, the included test cases cover a very wide range of possible input parameters.